theorems/T000297.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: T000297
+if:
+  P000138: true
+then:
+  P000057: true
+refs:
+  - mo: 418619
+    name: Is there an infinite topological space with only countably many continuous functions to itself?
+---
+
+Each constant map is a continous map.
```

Comment:
typo: continuous

Reply:
I think you should be able to edit typos directly in the GitHub web interface since my PR is a branch that lives in the repo. Feel free to do that directly (or let me know if you're unable to).

Reply:
> I think you should be able to edit typos directly in the GitHub web interface since my PR is a branch that lives in the repo. Feel free to do that directly (or let me know if you're unable to).

I am not sure is I can do it.  Went to https://github.com/pi-base/data/compare/master...mo418619, but when I go to the drop down menu for that file, the option "Edit file" is greyed out. Or am I supposed to try something else?

Reply:
Nah that means you probably don't have permission. I'll investigate (but leaving office for now).

Reply:
Anyway, modulo this typo I'm approving this.